data/sql_updates/communication_cost.sql

```diff
@@ -3,26 +3,30 @@ create materialized view ofec_communication_cost_mv_tmp as
 select
     row_number() over () as idx,
     *,
+    s_o_cand_id as cand_id,
+    org_id as cmte_id,
     report_pdf_url(image_num) as pdf_url
-from communication_costs_vw
-where rpt_yr >= :START_YEAR
+from nml_form_76
+where extract(year from communication_dt)::integer >= :START_YEAR
 ;
 
 create unique index on ofec_communication_cost_mv_tmp (idx);
 
 create index on ofec_communication_cost_mv_tmp (cmte_id);
 create index on ofec_communication_cost_mv_tmp (cand_id);
-create index on ofec_communication_cost_mv_tmp (cand_office_st);
-create index on ofec_communication_cost_mv_tmp (cand_office_district);
-create index on ofec_communication_cost_mv_tmp (cand_office);
-create index on ofec_communication_cost_mv_tmp (cand_pty_affiliation);
-create index on ofec_communication_cost_mv_tmp (transaction_dt);
-create index on ofec_communication_cost_mv_tmp (transaction_amt);
+create index on ofec_communication_cost_mv_tmp (s_o_cand_office_st);
+create index on ofec_communication_cost_mv_tmp (s_o_cand_office_district);
+create index on ofec_communication_cost_mv_tmp (s_o_cand_office);
+create index on ofec_communication_cost_mv_tmp (s_o_ind);
+--create index on ofec_communication_cost_mv_tmp (cand_pty_affiliation);
+--create index on ofec_communication_cost_mv_tmp (transaction_dt);
+--create index on ofec_communication_cost_mv_tmp (transaction_amt);
+create index on ofec_communication_cost_mv_tmp (communication_dt);
+create index on ofec_communication_cost_mv_tmp (communication_cost);
```

Comment:
I am just thinking of renaming it in the sql to keep it consistent with what we had. `communication_cost` in this table is the same as `transaction_amount` in the old table.